### ISSUE TITLE
Invalid or unsupported Type AWS::AppSync::GraphQLSchema

### ISSUE THREAD

**Issue by TLaue** [aws-cloudformation/cfn-python-lint#1372]
*cfn-lint version: (`cfn-lint --version`)*
cfn-lint 0.28.1
*Description of issue.*
Using the following CFN resource leads to the above mentioned linting error:
```
  D2VApiSchema:
    Type: AWS::AppSync::GraphQLSchema
    Properties:
      ApiId: !GetAtt D2VApi.ApiId
      DefinitionS3Location: Graphql/schema.graphql
```
Please provide as much information as possible:
* Template linting issues:
  * Please provide a CloudFormation sample that generated the issue.
  * If present, please add links to the (official) documentation for clarification.
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-appsync-graphqlschema.html
  * Validate if the issue still exists with the latest version of `cfn-lint` and/or the latest Spec files
* Feature request:
  * Please provide argumentation about the missing feature. Context is key!

Cfn-lint uses the [CloudFormation Resource Specifications](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-resource-specification.html) as the base to do validation. These files are included as part of the application version. Please update to the latest version of `cfn-lint` or update the spec files manually (`cfn-lint -u`)

**Comment by PatMyron** [comment]
[`AWS::AppSync::GraphQLSchema`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-appsync-graphqlschema.html) is in the latest [Resource Specification](https://d1uauaxba7bl26.cloudfront.net/latest/gzip/CloudFormationResourceSpecification.json), and I'm unable to reproduce this error using cfn-lint 0.28.1 with this template:
```yaml
Resources:
  D2VApiSchema:
    Type: AWS::AppSync::GraphQLSchema
    Properties:
      ApiId: !GetAtt D2VApi.ApiId
      DefinitionS3Location: Graphql/schema.graphql
```
Commands like these will update your Linter version and Resource Specifications:
```shell
pip3 install cfn-lint --upgrade
cfn-lint --update-specs
```

---

If you're still experiencing this error with cfn-lint 0.28.1, is there a particular region you're linting with? I'm only able to reproduce this in some regions other than the [`us-east-1`](https://d1uauaxba7bl26.cloudfront.net/latest/gzip/CloudFormationResourceSpecification.json) default

**Comment by TLaue** [comment]
I am using the following linter configuration:

Metadata:
cfn-lint:
    config:
      regions:
        - eu-central-1
      ignore_checks:
        - W2001 # ignore warnings about unused parameters
        - W7001 # ignore warnings about unused mappings

I am working in AWS region "eu-central-1" where this problem seems to occurr.

**Comment by PatMyron** [comment]
[`AWS::AppSync::GraphQLSchema`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-appsync-graphqlschema.html) is indeed not in the [`eu-central-1` Resource Specification](https://d1mta8qj7i28i2.cloudfront.net/latest/gzip/CloudFormationResourceSpecification.json)

Are you able to successfully deploy that resource type in `eu-central-1`?

**Comment by TLaue** [comment]
I am able to successfully deploy that resource type in eu-central-1.

**Comment by PatMyron** [comment]
[`AWS::AppSync::GraphQLSchema`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-appsync-graphqlschema.html) is now in the [`eu-central-1` Resource Specification](https://d1mta8qj7i28i2.cloudfront.net/latest/gzip/CloudFormationResourceSpecification.json):

```shell
cfn-lint --update-specs
```
